src/core/Akka.Persistence.TestKit/IJournalWriteInterceptor.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Akka.Persistence.TestKit
+{
+    using System.Threading.Tasks;
+
+    public interface IJournalWriteInterceptor
```

Comment:
I like this idea - having a generic action that can be used to intercept a common journal operation, like a Recover or Write.